people/2015/spring/mellolikejello.yaml

```diff
@@ -22,6 +22,7 @@ hw:
   commarchpreso: https://mellolikejello.wordpress.com/2015/03/11/commarch-presentation/
   curriculum: https://mellolikejello.wordpress.com/2015/03/18/curriculum/
   teamprop2: https://mellolikejello.wordpress.com/2015/03/20/anglegators-team-proposal/
+  projteam: amm4108, RedTwo
```

Comment:
I think that this should be a link to the blog post which contains your team. @decause @liam-middlebrook am I correct here?